scripts/projectView.js

```diff
@@ -10,29 +12,28 @@ projectView.handleMainNav = function() {
   $('.main-nav .tab:first').click();
 };
 
+//HIDES ALL BUT THE BEGINNING OF EACH PROJECT, SHOWS ALL WHEN CLICKED AGAIN
```

Comment:
This is a really cool feature. I like that you applied the setTeaser functionality to your past projects, with the read-less see-more toggle.